src/Formik.tsx

```diff
@@ -483,7 +484,11 @@ export class Formik<Values = FormikValues> extends React.Component<
       }));
 
       if (this.props.validateOnBlur) {
-        this.runValidations(this.state.values);
+        this.runValidations(this.state.values).then(combinedErrors => {
+          if (this.props.onBlur && field) {
```

Comment:
Do you know when there wouldn't be a `field` here?

Reply:
Not in my head, will test a bit

Reply:
Actually this is quite simple, you can have focused elements inside the form other than inputs (divs with tabIndex=0 for instance) which make the field not always present

Reply:
:+1: